include/zenoh-pico/protocol/definitions/transport.h

```diff
@@ -477,6 +477,8 @@ typedef struct {
 } _z_t_msg_fragment_t;
 void _z_t_msg_fragment_clear(_z_t_msg_fragment_t *msg);
 
+#define _Z_FRAGMENT_HEADER_SIZE 12
```

Comment:
This is actually not true, the fragment header size may vary depending on the used protocol extension.
As a consequence, basing the code logic on this value may lead to some bugs.

Reply:
I'd say we should fix this in another PR, I opened an issue on the overall issue: #295